fuzzylite/CMakeLists.txt

```diff
@@ -242,38 +269,23 @@ if(NOT FL_INSTALL_LIBDIR)
     endif()
 endif()
 
-if(FL_BUILD_BINARY)
-    install(TARGETS fl-bin
-            RUNTIME DESTINATION ${FL_INSTALL_BINDIR}
-            LIBRARY DESTINATION ${FL_INSTALL_LIBDIR}
-            ARCHIVE DESTINATION ${FL_INSTALL_LIBDIR}
-    )
-endif()
-
-if(FL_BUILD_SHARED)
-    install(TARGETS fl-shared
-            RUNTIME DESTINATION ${FL_INSTALL_BINDIR}
-            LIBRARY DESTINATION ${FL_INSTALL_LIBDIR}
-            ARCHIVE DESTINATION ${FL_INSTALL_LIBDIR}
-    )
-endif()
-
-if(FL_BUILD_STATIC)
-    install(TARGETS fl-static
-            RUNTIME DESTINATION ${FL_INSTALL_BINDIR}
-            LIBRARY DESTINATION ${FL_INSTALL_LIBDIR}
-            ARCHIVE DESTINATION ${FL_INSTALL_LIBDIR}
-    )
-endif()
+install(TARGETS ${fl-targets} EXPORT fuzzylite-config
+    RUNTIME DESTINATION ${FL_INSTALL_BINDIR}
+    LIBRARY DESTINATION ${FL_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${FL_INSTALL_LIBDIR}
+)
+install(EXPORT fuzzylite-config DESTINATION share/fuzzylite/cmake)
```

Comment:
is `share/fuzzylite/cmake` the default location for everything? Does it work on windows? How?

Reply:
`share/<project-name>/cmake` is the standard UNIX location for CMake configuration files. AFAIK, CMake doesn't support `install` on Windows at all, hence no Windows-specific CMake path.